pychord/analyzer.py

```diff
@@ -58,7 +58,17 @@ def get_all_rotated_notes(notes: List[str]) -> List[List[str]]:
 
     get_all_rotated_notes([A,C,E]) -> [[A,C,E],[C,E,A],[E,A,C]]
     """
-    notes_list = []
-    for x in range(len(notes)):
-        notes_list.append(notes[x:] + notes[:x])
-    return notes_list
+    if len(notes) == 0:
+        return []
+    if len(notes) == 1:
+        return [notes]
+    if len(notes) == 2:
+        return [[notes[0], notes[1]], [notes[1], notes[0]]]
+    list_of_chords = []
+    for i in range(0, len(notes)):
+        m = notes[i]
+        remaining_list = notes[:i] + notes[i + 1:]
+        permutations_on_remaining_list = get_all_rotated_notes(remaining_list)
```

Comment:
I don't think these are regular rotations of a chord. We should keep the backward compatibility for the `get_all_rotated_notes` function if it is not a bug.
Why do you need this modification? If you think this feature is useful for others, I recommend you create a new function or add a new argument to enable this.